distributed/cli/tests/test_dask_worker.py

```diff
@@ -493,3 +493,27 @@ def worker_type(dask_worker):
 
                 worker_types = await c.run(worker_type)
                 assert all(name == "MyWorker" for name in worker_types.values())
+
+
+@pytest.mark.asyncio
```

Comment:
Please use ``@gen_cluster(nthreads=[], client=True)`` instead

Reply:
I agree that our default should be to use `gen_cluster` / `gen_test`, but in this particular case the tests added are to specifically target that the `dask-worker` / `dask-scheduler` CLIs work as expected. Our `gen_*` utilities use the `Scheduler` / `Worker` classes directly, which don't have the same issue originally reported in https://github.com/dask/distributed/pull/4511

Reply:
But you're not using the dask-scheduler CLI.

Reply:
Whoops, I got mixed up and thought your comment was about the test added in `test_dask_scheduler.py`. You're totally right about about using `gen_cluster` on the worker test. Just pushed a commit to update.